### DISCUSSION TITLE
List of missing N64 CRC's

### DISCUSSION THREAD

**Discussion by majorpbx** [sanni/cartreader#214] [Ideas & Suggestions]
Since I'm running through several N64 titles and have found a few missing. I pulled a current missing list comparing to DoM and have come up with the following entries that are missing from "n64.txt" file. There are only 2 that I have no idea as to the save type. This should catch the "n64.txt" file up with current DoM status.

d5051096,NAYE,32,1
20fd2939,NYWJ,16,1
9181c1b7,NYWJ,16,1
7e74eedc,NHAJ,12,5
2d118764,NBYI,12,0
274d3493,ND3E,12,0
7ff42fd0,NCTE,12,5
08287cc8,NV2J,12,0
dd4e84e4,NK2J,16,5
ebaed1f9,NCWP,12,6
43ee0117,NDFJ,32,0
b566fb94,NDYJ,12,5
b9c9f07a,NDNF,08,0
e9cf3d9f,NWID,32,0
24d937bf,NETJ,16,0
1f4e651a,NFWP,12,5
68fe1cec,CFZE,32,1
c05b9184,NFRE,16,0
b3a034d6,NKAF,12,5
e373fa31,NHXF,08,0
c0ba9440,NNSX,16,0
dbd96deb,NIMJ,12,0
8a16a6a9,NISY,16,0
4c433fdd,NISE,16,0
7d0eed6a,NJEJ,08, - J.League Eleven Beat 1997 (Japan)
6fc0a31b,NGSJ,16,0
2ef647d3,NPSJ,16,4
f96c2be1,NPSJ,16,4
bc7ef5c1,NPEJ,16,0
c02fd9be,NJ5J,16,0
7994c2fe,NJ5J,16,0
1e53a7ba,NP6J,16,0
7ab7ebf4,NP6J,16,0
fcc10356,NP6J,16,0
e2e6823d,NZSP,32,4
04ea55ea,NZSP,64,4
5d1b2996,NZLP,64,1
56769a44,N9ME,12,0
a5071a77,NMFP,32,1
2fc4c216,NMFE,32,1
bb03a1a6,NMFJ,32,1
ad1758ac,NMFJ,32,1
176b3683,NMPJ,16,0
c3300cef,NTME,08,5
e1c470fd,NMIS,12,5
ba64f824,NMKE,12,0
b591cfea,NBIE,16,0
d175397b,NCED,32,0
16414ae4,NUTJ,16,0
f0bcd1ce,NUMJ,32,1
410f6510,NOBE,40,1
991f569c,NOBJ,40,1
9fc00ce3,NMQP,48,4
df56e922,NPFI,16,4
36ee5ca8,NPUP,12,0
b9be7b90,NKMJ,08,0
35461699,NNRJ,08, - Pro Mahjong Tsuwamono 64 - Jansou Battle ni Chousen (Japan)
832ea1da,NREE,64,1
886c5753,NSFE,08,0
cdf86f5e,NSYE,16,0
61e6642c,NSDB,32,0
87f00472,NSGY,16,0
9f74a58c,NSGJ,16,0
235caec6,NDTB,16,0
e32d9e7b,NSVE,08,5
65a8fba0,NRSE,16,5
bbf7b6a8,NR6F,16,0
62a8ce7d,N3TE,16,0
c4bc4df8,NGRX,08,0
c81f3321,NTHE,12,0
d1906de4,NTHD,12,0
2765dd8f,NTUD,08,0
72ca307a,NTUD,08,0
e28756f4,NRWE,08,0
57f1fbf5,NT2E,32,0
54d190a8,NWRJ,08,5
927383bb,NWDJ,16,0
e88d7e16,NW4P,32,4

**Comment by mauruco** [comment]
@majorpbx , what are the values ​​after CRC (NW4P,32,4) and how did you find them? I'm trying to learn as much as possible.
Do you know if it is possible to find out if a cartridge is a clone already through the display? No need to open it.

**Reply by majorpbx** [reply]
the values after the CRC goes like below. I am just learning to add code to the cart reader but if you were to use DoM DAT's it may be possible to find the hierarchy. The one's I've listed are either new additions, different revisions, or different region dumps.

CRC32,Cart Serial,Size in MB,Save Type

**Reply by mauruco** [reply]
Thanks. I found these two here (-> b3a034d6,KNAUF,12.5 -> bbf7b6a8,NR6F,16,0) in the current list, I don't know if it helps.

**Comment by sanni** [comment]
If you want you can make a Pull Request or I can just copy&paste them into n64.txt myself. The ones where you are unsure we can just set to save type 0.

**Reply by majorpbx** [reply]
Okay, I'll try a pull request and that's exactly what I did but I'm still trying to find out what save types they are to be sure.
